From c47b26d6b325577fbb7252ed2efdd591e8fc1af1 Mon Sep 17 00:00:00 2001
From: Kathiravan Thirumoorthy <quic_kathirav@quicinc.com>
Date: Fri, 4 Oct 2024 16:22:39 +0530
Subject: [PATCH] remoteproc: enable TMELCOM only for IPQ5424

Only IPQ5424 supports TMELCOM, so lets not enable for other targets

Change-Id: I68b5ad4ef07cafed9c719076127ad7d16adbf3b8
Signed-off-by: Kathiravan Thirumoorthy <quic_kathirav@quicinc.com>
---
 drivers/remoteproc/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/remoteproc/Kconfig b/drivers/remoteproc/Kconfig
index 772e0427f8a6..a03e5abe8e7c 100644
--- a/drivers/remoteproc/Kconfig
+++ b/drivers/remoteproc/Kconfig
@@ -299,7 +299,7 @@ config QCOM_Q6V5_WCSS_SEC
 	select QCOM_RPROC_COMMON
 	select QCOM_SCM
 	select TMEL_QMP
-	select QCOM_TMELCOM
+	select QCOM_TMELCOM if ARCH_IPQ5424
 	help
 	  Say y here to support the Secure Qualcomm Peripheral Image Loader
 	  for the Hexagon V5 based WCSS remote processors on e.g. IPQ54xx.
-- 
2.34.1

